<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MatchWindow</class>
 <widget class="QMainWindow" name="MatchWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1115</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MatchWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>971</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="turn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Turn: Player 1</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>589</y>
      <width>341</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="player1">
       <property name="text">
        <string>Player 1: 0 Points</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="player2">
       <property name="text">
        <string>Player 2: 0 Points</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>590</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="card_label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>590</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="cardVal_label">
       <property name="text">
        <string>Card Worth: --</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1115</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMain_Board">
    <property name="title">
     <string>Main Board</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuMain_Menu">
    <property name="title">
     <string>Main Menu</string>
    </property>
   </widget>
   <addaction name="menuMain_Board"/>
   <addaction name="menuMain_Menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
